lib/data_exporter.rb

```diff
@@ -1,13 +1,14 @@
 class DataExporter
   CSV_HEADERS = %i(id title count).freeze
 
-  EU_COUNTRIES = %w(
+  EUROPEAN_COUNTRIES = %w(
     austria belgium bulgaria croatia cyprus czech-republic denmark estonia finland france germany greece hungary
     ireland italy latvia lithuania luxembourg malta netherlands poland portugal slovakia slovenia spain sweeden
```

Comment:
Is Sweden intentionally meant to be spelt like `sweeden` here?

Reply:
Nope! Good catch, I'll raise a PR for it now.